libqtile/backend/wayland/core.py

```diff
@@ -1286,26 +1287,19 @@ def unhide_cursor(self) -> None:
     @expose_command()
     def get_inputs(self) -> dict[str, list[dict[str, str]]]:
         """Get information on all input devices."""
-        info = {}
-        device_lists: dict[str, list[inputs._Device]] = {
-            "type:keyboard": self.keyboards,  # type: ignore
-            "type:pointer": self._pointers,  # type: ignore
```

Comment:
This pr fixes something because pointer was hardcoded here, but we still need to do additional checking if it's a pointer or a touchpad. See https://github.com/qtile/qtile/blob/3225a562e0b0d573294dc140631399076a018d1f/libqtile/backend/wayland/inputs.py#L172

Reply:
I'm not sure I understand what would be left to fix - from what I can tell, your changes fix the issue.

Perhaps you mean that we don't report all possible input devices, such as 'touch' devices, as you mention in the OP re https://github.com/qtile/qtile/pull/3931? For a bit more context, we don't support those devices yet, but when we implement support we will have to ensure `get_inputs()` is reporting things as expected. I think your solution here wouldn't need changing as it is indifferent, but we'll probably end up needing a new `_Device` subclass like `Touch` that reports the correct info from `get_info()`.

Reply:
Yes that's what I was saying indeed.  The only thing we would then need to do is also add the touch devices to the list that we loop through in get_inputs.

My PR description was more of a thinking out loud as I was comparing the output to sway's output